client/ionic/src/App.tsx

```diff
@@ -38,21 +38,17 @@ const App: React.FC = () => (
     <IonReactRouter>
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
-        <Route
-          path="/coping-everyone"
-          component={CopingEveryone}
-          exact={true}
-        />
-        <Route path="/coping-parents" component={CopingParents} exact={true} />
+        <Route path="/splash-title" component={SplashTitle} exact={true} />
```

Comment:
@viviancromwell can you clarify the intention of the "Getting Started - first time user" flow? How does the user navigate through that?

@myiremark general comment that I'm leaving here is that I'm guessing user press on the card isn't how you navigate through, but will let Vivian speak to it